### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by davidbrochart**

### CODE REVIEW

**Comment by davidbrochart** [comment]
Not sure about the failing test, I just changed the documentation.
```
FAIL: test_bad_values (traittypes.tests.test_traittypes.TestIntArray)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/ipython/traitlets/traitlets/tests/test_traitlets.py", line 1157, in test_bad_values

    self.assertRaises(TraitError, self.assign, value)

AssertionError: TraitError not raised by assign

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "/home/travis/build/ipython/traitlets/traitlets/tests/test_traitlets.py", line 1159, in test_bad_values

    assert False, value

AssertionError: [1, [0, 0]
```

File: docs/source/config.rst [hunk 1]
```diff
@@ -295,24 +295,24 @@ and no spaces.
 
 .. note::
 
-    By default any error in configuration files with lead to this configuration
-    file be ignored by default.  Application subclasses may specify
+    Any error in configuration files wich lead to this configuration
```
Comment (jasongrout) [inline docs/source/config.rst]:
```suggestion
    Any error in configuration files which lead to this configuration
```

**Comment by jasongrout** [comment]
Thanks! One more typo correction, and then it's +1 from me!

**Comment by davidbrochart** [comment]
:smile: making typos trying to fix typos... I also forgot the verb.

**Comment by Carreau** [comment]
failure is unrelated. 

Thanks, merging.

**Comment by Carreau** [comment]
@meeseeksdev backport to 4.3.x

**Pull request merged**